roborock/local_api.py

```diff
@@ -51,6 +51,8 @@ async def keep_alive_func(self, _=None):
             await self.ping()
         except RoborockException:
             pass
+        if self.keep_alive_task is not None:
+            self.keep_alive_task.cancel()
```

Comment:
Is this needed? The task will end after next line

Reply:
My understanding is that call later returns a obj that can cancel the original callback, but when you call call_later it will run the action regardless of if the response gets overwritten.

I could be wrong though, I Didn’t extensively test

Reply:
Yes. But this is cancelling it during execution. Right after ping was already called and next one is being scheduled
I think there is a cancel already when disconnected

Reply:
That's the case if the keep_alive_func is called by the call_later in keep alive func, but if keep_alive_func is called in async connect, then the existing keep alive task still exist and will continue to populate itself, but the new one will also be populating itself.

I just removed it for now though because I am not confident.